modules/mono/glue/GodotSharp/GodotSharp/Core/Bridge/ScriptManagerBridge.cs

```diff
@@ -1064,16 +1066,16 @@ public void Dispose()
 
                 // There's no recursion here, so it's ok to go with a big enough number for most cases
                 // stackMaxSize = stackMaxLength * sizeof(godotsharp_property_info)
```

Comment:
I missed this one before.

```suggestion
                // StackMaxSize = StackMaxLength * sizeof(godotsharp_property_info)
```